guides-and-standards/guide-code-review.md

```diff
@@ -0,0 +1,144 @@
+
+In other cases, a reviewer may approve your PR during their initial review while also making comments or suggestions. This usually indicates that the reviewer felt their comments were minor or inessential and it is up to you to address them at your discretion before merging.
+
+### What if I have an approving review but other reviewers haven't finished reviewing yet?
+
+In general, once your PR has one approving review, you may choose to merge your PR without waiting for additional reviews. Note, however, that this is context-dependent. For example, you may want to wait longer for a review from someone who has domain expertise that is particularly relevant to your PR. Or, if your PR is large and complex, it is usually prudent to wait for at least two reviews before merging, rather than relying on just one reviewer to catch all potential issues.
+
+## For PR reviewers
+
+### What should I look for when reviewing a PR?
+
+In general, focus on reviewing the areas or aspects of the PR that you feel you're best positioned to review or that other reviewers are less likely to review. For reference, some of the most essential general areas to consider when reviewing PRs at Arcadia are listed below. These areas are distilled from the requirements listed in the [standards for public repos](standard-public-repos.md); refer to that document for more details.
+
+- **Correctness**: Does the code do what it is supposed to do? Are there any bugs or errors?
+- **Documentation**: Is the code well-documented? Are there clear instructions for how to use the code?
+- **Dependencies**: Are all dependencies listed? Are there any dependencies that are not necessary or that could be removed?
+- **Security**: Are there any security vulnerabilities in the code? Is sensitive information handled securely?
+- **Error handling**: Is the code sufficiently robust in the face of errors?
+- **Impact**: Are there any potential side effects of the changes that are not immediately obvious?
+
+Depending on the nature and scope of the repository, areas related to code quality may also be important to review. Generally, these areas are less important for analysis code or one-off scripts and more important for code that is likely to be used or developed over time and/or depended on by other users. These areas include:
+
+- **Clarity**: Is the code readable and easy to understand? Are variable names descriptive? Are comments used where necessary?
+- **Consistency**: Does the code follow the same style and conventions as the rest of the project?
+- **Performance**: Is the code efficient? Are there any bottlenecks or performance issues that could easily be improved?
+- **Testing**: Are there tests for critical parts of the code? Do the tests cover a reasonable set of possible cases? Are there sufficiently realistic test datasets?
+
```

Comment:
Something to the effect of below. I'm not in love with my phrasing/grammar but I did my best. 

```suggestion
You may sometimes be tagged to review a PR that you don't feel you have the expertise to review in all of the capacities listed above. It's ok to review the components you feel comfortable commenting on and to add a comment to your review stating what you were able to give feedback on. If you were tagged to review a specific portion of the PR and you were able to review it, you can accept the changes if they are of sufficient quality. If you were tagged to review the entire PR and it is outside your scope of expertise, it's still worth it to try to review the code and leave comments on things that catch your attention. Then, you can communicate with the person who opened the PR to find another reviewer for the code (if they haven't already requested another reviewer). This is a great way to learn more about a code base and to learn more about code review.
```

Reply:
Great point! I've added this in a new section "what if I don't feel qualified to review a PR?"